packages/inter-protocol/scripts/start-local-chain.sh

```diff
@@ -55,7 +55,7 @@ make scenario2-setup-nobuild >>"$CHAIN_LOG" 2>&1
 echo "Starting the chain..."
 # use -economy target to get the kitchen sink
 # disable pruning to keep all history https://docs.desmos.network/fullnode/overview/
-make AGC_START_ARGS="--pruning=nothing" CHAIN_BOOTSTRAP_VAT_CONFIG=@agoric/boot/decentral-itest-vaults-config.json scenario2-run-chain >>"$CHAIN_LOG" 2>&1 &
+make AGC_START_ARGS="--pruning=nothing" CHAIN_BOOTSTRAP_VAT_CONFIG=@agoric/vm-config/decentral-itest-vaults-config.json scenario2-run-chain >>"$CHAIN_LOG" 2>&1 &
```

Comment:
Does. this (and the reference in `test-proposal-stuff.js`) mean that `inter-protocol/package.json` should take a dependency on `@agoric/vm-config`?

Reply:
I don't think so, because this is instructing `cosmic-swingset` what to do. That's the package that really has the dependency. Same as how the JSONs in vm-config that specify a slate of packages doesn't itself depend on them (the thing that reads their source does).

Also this script is not long for this world https://github.com/Agoric/agoric-sdk/pull/7914